client/my-sites/plugins/controller.js

```diff
@@ -91,16 +92,17 @@ function renderPluginList( context, basePath ) {
 	lastPluginsQuerystring = context.querystring;
 	context.store.dispatch( setTitle( i18n.translate( 'Plugins', { textOnly: true } ) ) ); // FIXME: Auto-converted from the Flux setTitle action. Please use <DocumentHead> instead.
```

Comment:
Sure! I thought this is outside of this PR's scope, but it's fine to address it here, too.